ui/src/app/pages/access/controlled-tier-card.tsx

```diff
@@ -93,13 +93,13 @@ const ControlledTierStep = (props: {
 
 export const ControlledTierCard = (props: {
   profile: Profile;
-  activeModule: AccessModule;
-  clickableModules: AccessModule[];
+  focusedModule: AccessModule;
+  activeModules: AccessModule[];
```

Comment:
The modules that were able to be interacted with used to be called `clickableModules`, but this naming was confusing, because it implied that the module itself can be clicked on. This is not always the case.